src/gleam/http/request.gleam

```diff
@@ -284,3 +285,21 @@ pub fn get_cookies(req) -> List(#(String, String)) {
   })
   |> list.flatten()
 }
+
+/// Remove a cookie from a request
+///
+/// Remove a cookie from the request. If no cookie is found return the request unchanged.
+pub fn remove_cookie(req: Request(body), name: String) {
+  case list.key_pop(req.headers, "cookie") {
+    Ok(#(cookies_string, headers)) -> {
+      let new_cookies_string =
+        string.split(cookies_string, ";")
+        |> list.map(string.trim)
+        |> list.filter(fn(str) { string.starts_with(str, name) |> bool.negate })
```

Comment:
In case you didn't know, you can use `!` rather than `bool.negate`, it's a bit shorter.